int/tests/conftest.py

```diff
@@ -8,23 +8,31 @@
 from acapy_backchannel.models.conn_record import ConnRecord
 import pytest
 import hashlib
+import httpx
 
 from acapy_backchannel import Client
 from acapy_backchannel.api.connection import (
     create_static,
     set_metadata,
     delete_connection,
 )
+from acapy_backchannel.api.wallet import create_did
+from acapy_backchannel.api.ledger import accept_taa, fetch_taa
 from acapy_backchannel.models import (
     ConnectionStaticRequest,
     ConnectionStaticResult,
     ConnectionMetadataSetRequest,
+    TAAAccept,
 )
 
 from aries_staticagent import StaticConnection, Target
 
 from . import BaseAgent
 
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Minor quip, `LOGGER` is preferable to `logger` since it's a module level global variable :slightly_smiling_face: